pkg/scheduler/ugm/manager.go

```diff
@@ -477,13 +478,14 @@ func (m *Manager) clearEarlierSetUserLimits(newUserLimits map[string]map[string]
 // resetUserEarlierUsage Clear or reset earlier usage only when user already tracked for the queue path.
 // Reset the max apps and max resources to default, unlink the end leaf queue of queue path from its immediate parent and
 // eventually remove user tracker object itself from ugm if it can be removed.
-func (m *Manager) resetUserEarlierUsage(ut *UserTracker, hierarchy []string) {
+func (m *Manager) resetUserEarlierUsage(ut *UserTracker, queuePath string) {
 	// Is this user already tracked for the queue path?
+	hierarchy := strings.Split(queuePath, configs.DOT)
 	if ut.IsQueuePathTrackedCompletely(hierarchy) {
 		log.Log(log.SchedUGM).Debug("Need to clear earlier set configs for user",
 			zap.String("user", ut.userName),
 			zap.Strings("queue path", hierarchy))
-		ut.setLimits(hierarchy, nil, 0, false, false)
+		ut.clearLimits(queuePath)
```

Comment:
Same comment

Reply:
In both cases, `useWildCard` = false, is that correct?

Reply:
Correct, but 5th argument differs for a reason. These changes has been added very recently, might lack some test coverage. You can file a jira for unit test coverage later, but not for the actual code changes. We should retain the behaviour as is to avoid unnecessary issues.